run-vmtest/run.sh

```diff
@@ -3,14 +3,28 @@
 set -euo pipefail
 trap 'exit 2' ERR
 
-source $(cd $(dirname $0) && pwd)/../helpers.sh
+source "${GITHUB_ACTION_PATH}/../helpers.sh"
+
+RUN_BPFTOOL_CHECKS=
+
+if [[ "$KERNEL_TEST" != "sched_ext" ]]; then
+	VMTEST_SCRIPT="${GITHUB_ACTION_PATH}/../ci/vmtest/vmtest_selftests.sh"
+	if [[ "${KERNEL}" = 'LATEST' ]]; then
+		RUN_BPFTOOL_CHECKS=true
+	fi
+else
+	VMTEST_SCRIPT="${GITHUB_ACTION_PATH}/../ci/vmtest/sched_ext_selftests.sh"
+fi
+
```

Comment:
how about we use `KERNEL_TEST` has the key to find which vmtest script to use?
Probably worth renaming `vmtest_selftests.sh` into `bpf_selftests.sh`? And use this as a key?

Reply:
it seems you dealt with this in #150 :)